Dockerfile

```diff
@@ -1,28 +1,22 @@
-FROM alpine
-  
-# Setup apt
-RUN apk -U --no-cache add \
-               build-base \
-               git \
-               linux-headers \
-               python3 \
-               python3-dev && \
-# Setup Snare 
-    git clone --depth=1 https://github.com/mushorg/snare /opt/snare && \
-    cd /opt/snare/ && \
-    pip3 install --no-cache-dir --upgrade pip setuptools && \
-    pip3 install --no-cache-dir -r requirements.txt && \
-    python3.6 setup.py install && \
-    cd / && \
-    rm -rf /opt/snare && \
-    clone --target http://example.com && \
-# Clean up
-    apk del --purge \
-            build-base \
-            linux-headers \
-            python3-dev && \
+FROM python:3.6-alpine3.8
+
+ARG PAGE_URL=example.com
```

Comment:
Isn't it better to put ARG and ENV commands just before «RUN clone» to keep build context as full as possible?